### PR TITLE
test(integrate): ienvelope calculation with lcao

### PR DESCRIPTION
**Pull request by hongriTianqi**
ienvelope calculations were tested in 212_NO_wfc_ienvelope (multi-k case) and 312_NO_GO_wfc_ienvelope (gamma only case), in which LOWF files were read by a cpp file (tests/integrate/tools/sum_ENV_H2.cpp). sum_ENV_H2.cpp reads output files and calculates norm of the wavefunction, the results should be 1.0. Note that sum_ENV_H2.cpp is compiled by cmake and then installed back into tests/integrate/tools directory. For makefile users this might be a problem for absence of executable sum_ENV_H2.

### CODE REVIEW

File: tests/integrate/tools/sum_ENV_H2.cpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+{
+	string input_file = argv[1];
+
+	ifstream inp(input_file.c_str(), ios::in);
+
+	if (!inp) 
+	{
+        	cout << "\e[1;31m [  FAILED  ]  \e[0m";
+		cout << "Can't find " << input_file << " !" << endl;
+		return 0;
```
Comment (caic99) [inline tests/integrate/tools/sum_ENV_H2.cpp]: For linux convention, programs returns 0 for normal exit, and 1 for abnormal ones.

**Review by caic99 — APPROVED** [review]

File: tests/integrate/tools/sum_ENV_H2.cpp (hunk 1 above)
Comment (hongriTianqi) [reply tests/integrate/tools/sum_ENV_H2.cpp]: OK

**Review by dyzheng — CHANGES_REQUESTED** [review]
The files in OUT.autotest would be deleted by clean.sh , We should move it to path tests/integrate/xx/ , and update code to support it.

**Comment by hongriTianqi** [comment]
> The files in OUT.autotest would be deleted by clean.sh , We should move it to path tests/integrate/xx/ , and update code to support it.

done

**Review by dyzheng — APPROVED** [review]

**Comment by caic99** [comment]
This PR failed to compile, see [log](https://github.com/deepmodeling/abacus-develop/runs/6627071904?check_suite_focus=true). @hongriTianqi Would you fix it? This problem seems to be related with openmp.

**Comment by hongriTianqi** [comment]
> This PR failed to compile, see [log](https://github.com/deepmodeling/abacus-develop/runs/6627071904?check_suite_focus=true). @hongriTianqi Would you fix it? This problem seems to be related with openmp.

Did you see anything to do with openmp in the file sum_ENV_H2.cpp? @caic99

**Comment by caic99** [comment]
> > This PR failed to compile, see [log](https://github.com/deepmodeling/abacus-develop/runs/6627071904?check_suite_focus=true). @hongriTianqi Would you fix it? This problem seems to be related with openmp.
> 
> Did you see anything to do with openmp in the file sum_ENV_H2.cpp? @caic99

I figured it out, and I'll submit a patch for that.

**Comment by hongriTianqi** [comment]
> I figured it out, and I'll submit a patch for that.

Thanks a lot. @caic99

**Pull request merged**